src/main/scala/io/iohk/ethereum/jsonrpc/ExpiringMap.scala

```diff
@@ -55,6 +57,13 @@ class ExpiringMap[K, V] private (val underLaying: mutable.Map[K, ValueWithDurati
   }
 
   private def isNotExpired(value: ValueWithDuration[V]) =
-    Duration.between(value.expiration, Instant.now(clock)).isNegative
+    currentNanoDuration.minus(value.expiration).isNegative
+
+  private def currentPlus(duration: Duration) =
+    currentNanoDuration.plus(duration)
+
+  private def currentNanoDuration =
```

Comment:
Style: it's suggested to use parenthesis for side-effecting function (`def currentNanoDuration()`). This form suggests pure function.